packages/terra-progress-bar/src/ProgressBar.scss

```diff
@@ -1,4 +1,17 @@
 :local {
+
+  @mixin terra-gradient($from-color, $to-color) {
+    background:
+      linear-gradient(to left, transparent, rgba(255, 255, 255, 0.5)),
+      linear-gradient(to left, $from-color, $to-color);
```

Comment:
Need to remove `linear-gradient(to left, $from-color, $to-color);`. In testing, this line causes the gradient added in the `::ms-fill selector` to not render the currentColor as expected. All of the gradient progress bars appeared with green background + gradient. Removing this line `linear-gradient(to left, $from-color, $to-color);` fixed the issue in IE / Edge. I didn't notice any side effects of removing this in the other browsers.

Reply:
Thanks. removed `linear-gradient(to left, $from-color, $to-color);` and tested in all browsers.